### DISCUSSION TITLE
Test run in Cypress test runner not reflecting new changes to feature file

### DISCUSSION THREAD

**Discussion by jessica-jorgenson** [badeball/cypress-cucumber-preprocessor#1187] [Q&A]
Hello!
My team is experiencing the occasional issue where any changes they make to the feature file isn't reflected in the test when we're running `cypress open`. We haven't been able to capture anything on any DEBUG logs, or make a reproducible example, so I hesitate to make an issue.
Our current workaround is to make a differently named feature file, copy the steps over to that one, and run from that file. Once we see the new changes reflected in the runner, we delete the old file and rename the new feature file. We're thinking it might be an ESBuild issue, but it's a headache to change the preprocessor, and still not certain it's from there.

We've had these issues over several versions, but here's our current setup:
```
    "@badeball/cypress-cucumber-preprocessor": "^20.0.3",
    "@bahmutov/cypress-esbuild-preprocessor": "^2.2.0",
    "cypress": "^13.5.1",
    "ts-node": "^10.9.1",
    "typescript": "^5.2.2",
```
where we've followed the config setup for ESBuild TS documented [here](https://github.com/badeball/cypress-cucumber-preprocessor/tree/master/examples/esbuild-ts), though not using the latest versions for dependency issues for other plugins we're using.

Would anyone know what our issue could be?
Thanks!

**Comment by badeball** [comment]
Does [this](https://github.com/bahmutov/cypress-esbuild-preprocessor/issues/380) or [this](https://github.com/bahmutov/cypress-esbuild-preprocessor/issues/195) describe your issue and help you reproduce it? If so, then in `@bahmutov/cypress-esbuild-preprocessor` lies your issue.
